### PR TITLE
Description of the project added in 'readme.md' file

### PR DESCRIPTION
**Pull request by DipenStha**

### CODE REVIEW

File: README.md
Comment (subashbasnyat) [inline README.md:1]: @DipenStha Please add, how to run the flask app in the README too.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Resume Analyzer
 
-This project aims to develop a resume evaluation model to improve the hiring process by
-extracting and matching resume data to job descriptions.
+The Resume Analyzer is a machine learning project designed to evaluate resumes and provide insights such as skill matching, role fit, and automated feedback based on job descriptions. This tool helps recruiters and applicants streamline the hiring process.
+
+Features:s
```
Comment (subashbasnyat) [inline README.md]: Remove,  extra s

File: README.md
Comment (DipenStha) [reply README.md:1]: Ok boss

File: README.md (hunk 1 above)
Comment (DipenStha) [reply README.md]: I have already edited that and created a pull request

**Review by subashbasnyat — APPROVED** [review]

**Pull request merged**